service/JobRoleService.ts

```diff
@@ -0,0 +1,16 @@
+import { JobRole } from "../model/JobRole";
+
+const axios = require('axios');
+
+module.exports.getJobRoles = async function()
+{
+    try
+    {
+        const response = await axios.get('http://localhost:8080/api/job-roles')
+
+        return response.data
```

Comment:
Consider moving the base url out to an environment variable. Might make things a bit easier for when we come to deploying this to AWS

```suggestion
module.exports.getJobRoles = async function()
{
    try
    {
       const baseURL = process.env.baseURL
       
        const response = await axios.get(`${baseURL}/api/job-roles`)

        return response.data
```